.github/workflows/main-checks.yml

```
@@ -181,15 +181,23 @@ jobs:
 
       - name: Run native tests
         uses: actions-rs/cargo@v1
+        env:
+          # workaround for lack of ternary operator
+          # see https://github.com/orgs/community/discussions/25725
+          RUSTFLAGS: ${{ matrix.toolchain == 'nightly && '--cfg nightly_yew' || '' }}
```

```suggestion
          RUSTFLAGS: ${{ matrix.toolchain == 'nightly' && '--cfg nightly_yew' || '' }}
```

Oops, didn't find this during the review.